net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson vala xdg
+
+DESCRIPTION="GNOME framework for accessing online accounts"
+HOMEPAGE="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
```

Comment:
Please use https://gitlab.gnome.org/GNOME/gnome-online-accounts as wiki.gnome.org will be disabled soon